bliss/data_augmentation.py

```diff
@@ -0,0 +1,117 @@
+import random
+
+import numpy as np
+import torchvision as TF
+
+from bliss.catalog import TileCatalog
+
+
+def augment_data(tile_catalog, image, deconv_image=None):
+    origin_tile = TileCatalog(4, tile_catalog)
+    origin_full = origin_tile.to_full_params()
+
+    num_transform_list = [1, 2, 3]
+    num_transform = random.choices(num_transform_list, weights=[0.7, 0.2, 0.1], k=1)[0]
+    transform_list = ["vflip", "hflip", "rotate90", "rotate180", "rotate270", "shift"]
+
+    aug_method = np.random.choice(
+        transform_list, num_transform, p=(0.1, 0.1, 0.1, 0.1, 0.1, 0.5), replace=False
```

Comment:
We'd each of the eight rotation-flip pairs to be equally likely? Does this achieve that? Is there a simpler way to write it?

Reply:
Don't we always want to call shift? With probability 1/16 it will shift by (0,0), effectively leaving the image unchanged, right?

Reply:
I don't think the code as is gives us uniform coverage of rotation-flip pairs, which is what we want. It also seems a bit hard to understand.

This may be simpler way to do it: First, select one of four rotations (including 0 degrees -- a "no op") and apply. Next, select whether to flip the image vertically or not. (No need to every flip horizontally--that's the same as a vertical flip and a 180 degree rotation.) Finally, apply a random shift.

Reply:
Nice job with these changes